Makefile

```diff
@@ -15,7 +17,14 @@ vagrant-up:
 aws: aws-provision ansible-galaxy site
 
 aws-provision: 
-	ansible-playbook -i localhost, aws-provision.yml -v --ask-vault-pass
+	ansible-playbook -i localhost, aws-provision.yml -v
 
 site: 
-	ansible-playbook -i ec2.py site.yml -v --ask-vault-pass
+	ansible-playbook -i ec2.py site.yml -v
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys $(var);)
```

Comment:
Could you update this to match alphagov/tsuru-ansible@967c65e7621c36d0f4535a65a9ecd61a9dc36b43?

Might also be worth pulling in alphagov/tsuru-ansible#60, if that's going to get merged.